index.html

```diff
@@ -13,18 +13,30 @@
             <div class="nav_container">
                 <a href="#" class="logo">
                      <img src="images/MyLogo.png" alt="Logo">
-                </a>
-     
+                </a>    
                 <nav class="toggle">
                     <img src="images/Union.png" alt="hamburger menu">
                 </nav>
+                <nav class = "desktop_nav">
```

Comment:
- [ ] All over the project, the fonts that are being displayed are not the ones that you are declaring in your CSS (eg. 'Poppins').  This is because you have not imported the fonts from the source (google fonts).   Let's grab them from [here](https://fonts.google.com/specimen/Poppins#standard-styles).

Look:

![image](https://user-images.githubusercontent.com/57421823/149128993-9b074505-e598-4acc-9abe-5ae83f4d2871.png)

You need to choose ALL the font weights that you are going to need (400, 500, 700 are almost always needed).  After choosing them, you will get 3 HTML `<link>`s:

![image](https://user-images.githubusercontent.com/57421823/149129362-05a2578d-b93a-4742-ad9a-d73e83db2625.png)

Include them into the `<head>` of your HTML file (it's on the top of the file).

- [ ] Also, on all the section that includes your projects, in the parts that read: 'Back End Dev' and '2015', you are not declaring the fonts.  

![image](https://user-images.githubusercontent.com/57421823/149130042-72d0d2c8-4be6-4252-945e-7e2689a1d686.png)

Kindly correct them.

That way, your fonts will work!